src/core/parser.zsh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env zsh
 
 _fzf_complete_get_command_pos() {
+    local cmd
     local arguments=("${(Q)${(z)@}[@]}")
-    local cmd=$(__fzf_extract_command "$@")
+
+    if [[ "$(which __fzf_extract_command)" == *compstate* ]]; then
```

Comment:
Not the result of `fzf --version` but `$(which __fzf_extract_command) is used because there may be who are fetching only `completion.zsh` such as my dotfiles https://github.com/hexium310/dotfiles/blob/6b61cdf90e49bb08824dcd779370436189cf010e/config/zsh/.zshrc#L23 .